docs/data/joy/components/select/SelectFieldDemo.js

```diff
@@ -6,22 +6,30 @@ import Select from '@mui/joy/Select';
 import Option from '@mui/joy/Option';
 
 export default function SelectFieldDemo() {
-  const [value, setValue] = React.useState('dog');
   return (
     <FormControl sx={{ width: 240 }}>
-      <FormLabel
-        // screen readers will announce "Favorite pet, dog selected" when the select is focused.
-        aria-label={`Favorite pet, ${value} selected.`}
-      >
+      <FormLabel id="select-field-demo-label" htmlFor="select-field-demo-button">
```

Comment:
@oliviertassinari You can revert the changes for this demo, the `FormControl` has been integrated with `Select`.

Proof: https://mui.com/joy-ui/react-select/#accessibility

Reply:
See https://github.com/mui/material-ui/pull/34116#discussion_r973797825.